### DISCUSSION TITLE
Elasticsearch: Better defaults for rate aggregation

### DISCUSSION THREAD

**Discussion by Elfo404** [grafana/grafana#58407] [Data Sources]
follow up on @ifrost comment on #33311

It would be nice to have an `Auto` option for the `unit` setting for the `rate` aggregation to improve UX.

Rate Aggregation in ES: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-rate-aggregation.html

_Originally posted by @ifrost in https://github.com/grafana/grafana/pull/33311#discussion_r655374405_

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
